### PR TITLE
Add view

### PR DESCRIPTION
**Pull request by vodinhduy197**

### CODE REVIEW

**Comment by vodinhduy197** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -11,4 +11,4 @@
 |
 */
 
-Route::resource('/','Index');
+Route::resource('/task','Index');
```
Comment (tranducquoc) [inline routes/web.php]: index chứ sao Index hả em? i viết thường nhé

**Comment by tranducquoc** [comment]
commented

**Comment by vodinhduy197** [comment]
ready

File: app/Http/Controllers/Home.php [hunk 2]
```diff
@@ -87,6 +87,13 @@ public function update(Request $request, $id)
      */
     public function destroy($id)
     {
-        //
+        try{
```
Comment (thanhhungchu95) [inline app/Http/Controllers/Home.php]: add an extra space before braces

File: app/Http/Controllers/Home.php [hunk 3]
```diff
@@ -87,6 +87,13 @@ public function update(Request $request, $id)
...
+            $task = Task::findOrFail($id);
+            $task->delete();
+
+            return redirect("/task")->with('success', trans('index.delete-success'));
+        }catch (Exception $exception){
```
Comment (thanhhungchu95) [inline app/Http/Controllers/Home.php]: same at here

File: resources/lang/en/index.php [hunk 4]
```diff
@@ -1,15 +1,17 @@
 <?php
 return [
     'title' => 'Laravel Quickstart - Basic',
-    'label_task' => 'Tasks',
-    'btn_addTask' => 'Add Tasks',
+    'label_task' => 'Task',
+    'btn_addTask' => 'Add Task',
```
Comment (thanhhungchu95) [inline resources/lang/en/index.php]: use snake-case only

File: resources/views/message.blade.php [hunk 5]
```diff
@@ -22,3 +22,12 @@
         </button>
     </div>
 @endif
+
+@if(Session::has('fail'))
+    <div class="alert alert-danger alert-dismissible fade show col col-8" role="alert">
+        <strong>{{Session::get('fail')}}</strong>
```
Comment (thanhhungchu95) [inline resources/views/message.blade.php]: add extra space inside braces

**Comment by thanhhungchu95** [comment]
commented

**Comment by vodinhduy197** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**